tools/pot/tests/test_graph.py

```diff
@@ -277,7 +280,7 @@ def test_lstm_ends(tmp_path, models):
 
 
 TEST_MODELS_WITHOUT_FQ_MOVING = [
-    ('test_multibranch_propogation_with_fq_moving', 'pytorch')
+    ('test_multibranch_propogation_with_fq_moving', 'pytorch', )
```

Comment:
Is there any reason to have `,` at last?

Reply:
No reason, accidental comma, I'll remove it